shell/platform/android/io/flutter/embedding/engine/loader/FlutterLoader.java

```diff
@@ -78,9 +56,17 @@ public static FlutterLoader getInstance() {
     return instance;
   }
 
+  @NonNull
+  public static FlutterLoader getInstanceForTest(FlutterApplicationInfo flutterApplicationInfo) {
```

Comment:
I am not too happy about this but a test is relying on default values found in FlutterApplicationInfo to validate asset loading in FlutterLoader.